app/views/tasks/show.html.slim

```diff
@@ -504,23 +504,23 @@ legend.toggle-next
                   .row-value
                     = file.usage_by_lms
 .button-panel
-  /.button-box
+  .button-box
     .button-description
-      = "Save Exercises for later by adding them to your cart or a collection."
+      = "Save Tasks for later by adding them to your cart or a collection."
```

Comment:
Would you mind moving this part to the en.yml, please? It should be part of the localizable strings, not just included in the HTML template.

Reply:
Sure. I have added the corresponding translation in the en.yml file in the tasks.show section. Should I remove the corresponding ones from exercises.show?

Reply:
If it is not used somewhere else, yes please. If it is used somewhere else, please update the occurrences first and then proceed with removing it from the old path.

Reply:
Changes done and pushed :) Thanks for the feedback

Reply:
You're welcome -- thanks for incorporating it. Just two more things to go, I think, and then we should be quite ready 💪